### PR TITLE
unix-linux: Fix uv_resident_set_memory executable name

### PR DESCRIPTION
**Pull request by inPhraZ**
The filename of the executable may contain parentheses. If so, `*rss` will be invalid.

### CODE REVIEW

File: src/unix/linux.c [hunk 1]
```diff
@@ -1627,53 +1627,34 @@ uint64_t uv__hrtime(uv_clocktype_t type) {
 }
 
 
-int uv_resident_set_memory(size_t* rss) {
+int uv_resident_set_memory(size_t* rss)
+{
```
Comment (bnoordhuis) [inline src/unix/linux.c:1631]:
```suggestion
int uv_resident_set_memory(size_t* rss) {
```

File: src/unix/linux.c [hunk 2]
```diff
@@ -1627,53 +1627,34 @@ uint64_t uv__hrtime(uv_clocktype_t type) {
...
 
-  s = strchr(s, ')');
+  /* find the last ')' */
+  s = strrchr(buf, ')');
   if (s == NULL)
     goto err;
 
-  for (i = 1; i <= 22; i++) {
+  for (i = 1; i <= 22; i++)
+  {
```
Comment (bnoordhuis) [inline src/unix/linux.c:1649]:
```suggestion
  for (i = 1; i <= 22; i++) {
```

File: src/unix/linux.c [hunk 3]
```diff
@@ -1627,53 +1627,34 @@ uint64_t uv__hrtime(uv_clocktype_t type) {
...
     if (s == NULL)
       goto err;
   }
 
   errno = 0;
   val = strtol(s, NULL, 10);
-  if (errno != 0)
-    goto err;
-  if (val < 0)
+  if (val < 0 || errno != 0)
```
Comment (bnoordhuis) [inline src/unix/linux.c:1657]: I can't post this as a diff but please undo. Stylistic and functional changes shouldn't be intermixed in a single commit.

**Review by bnoordhuis — COMMENTED** [review]
LGTM but please fix the style issues.

File: src/unix/linux.c (hunk 3 above)
Comment (inPhraZ) [reply src/unix/linux.c:1657]: I separated the commit and reopened a new PR

**Pull request closed without merging**